README.md

```diff
@@ -38,8 +38,9 @@ instead of `New()`.
 shortuuid.NewWithNamespace("http://example.com")
 ```
 
-It's possible to use a custom alphabet as well, though it has to be 57
-characters long.
+It's possible to use a custom alphabet as well (at least 2
+characters long).  
+It will automatically sort and remove duplicates from your alphabet to ensure consistency
```

Comment:
In hindsight, duplicates should probably have been an error. Since we silently reduce the number of possibilities. But too late now since it's a breaking change.

Reply:
It is also how python's shortuuid library hadnles it, so I think it's better to keep it that way to have compatibility.
```python
>>> shortuuid.set_alphabet("aaaaabcdefgh1230123")
>>> shortuuid.get_alphabet()
'0123abcdefgh'
```